app/src/pages/inside/common/itemInfo/attributesBlock/attributesBlock.jsx

```diff
@@ -21,16 +21,33 @@ import styles from './attributesBlock.scss';
 
 const cx = classNames.bind(styles);
 
-export const AttributesBlock = ({ attributes, onClickAttribute, isAttributeClickable }) => (
+export const AttributesBlock = ({
+  attributes,
+  onClickAttribute,
+  isAttributeClickable,
+  hasHoverEffects,
+}) => (
   <div className={cx('attributes-block')}>
     {attributes.some(notSystemAttributePredicate) && <div className={cx('attributes-icon')} />}
     {attributes.filter(notSystemAttributePredicate).map((attribute) => (
       <div
         key={formatAttribute(attribute)}
-        className={cx('attribute', { cursor: isAttributeClickable })}
+        className={cx('attribute', {
+          cursor: isAttributeClickable,
+          'no-hover-effects': !hasHoverEffects,
+        })}
         onClick={isAttributeClickable ? () => onClickAttribute(attribute) : null}
+        title={formatAttribute(attribute, true)}
```

Comment:
In my opinion it is better to create two different functions, than pass true or false as argument.
It looks unclear and person who reading that code forced to open function definition.

Reply:
I'm not sure than it's correct to create another same function that differs only with ' : ' instead of ':'

Reply:
I would like to avoid refactoring of whole places that use `formatAttribute` so I can make another method that differs only with divider or left like it's done now